components/gitpod-db/src/user-to-team-migration-service.ts

```diff
@@ -72,9 +75,6 @@ export class UserToTeamMigrationService {
         if (!team) {
             throw new ResponseError(ErrorCodes.CONFLICT, "Could not create team for user.", { userId: user.id });
         }
-        // add membership
-        await this.teamDB.addMemberToTeam(user.id, team.id);
-        await this.teamDB.setTeamMemberRole(user.id, team.id, "owner");
```

Comment:
What's the reason for this removal?

Reply:
Sorry, wanted to add a comment. This is all done as part of createTeam